packages/orchestrator-ui-components/src/pages/startPage/queryVariables.ts

```diff
@@ -100,12 +101,19 @@ export const taskListSummaryQueryVariables: GraphqlQueryVariables<Process> = {
     ],
 };
 
-export const productsSummaryQueryVariables: GraphqlQueryVariables<ProductsSummary> =
-    {
-        first: 1000,
-        after: 0,
-        sortBy: {
-            field: 'name',
-            order: SortOrder.ASC,
+export const productsSummaryQueryVariables: GraphqlQueryVariables<
+    ProductsSummary & Subscription
```

Comment:
I'm not very familiar with the typescript type system, but having a union type of a ProductSummary and a Subscription feels a bit weird, since they are kind of unrelated. Unless this is just meant as a quick way to avoid introducing 'real' types. In that case you might be better of with something like (if that is even possible): 'type myCombinedWhateverType = ProductsSummary & Subscription' and use that

Reply:
In this query both are actually related: counting the number of subscriptions per product